model/task/task_test.go

```diff
@@ -2178,24 +2178,25 @@ func TestDeactivateTasks(t *testing.T) {
 	require.NoError(t, db.ClearCollections(Collection, event.EventCollection))
 
 	tasks := []Task{
-		{Id: "t0"},
+		{Id: "t0", DisplayOnly: true, ExecutionTasks: []string{"t6"}},
```

Comment:
Can we add a case where we check that a task that depends on the execution task `t6` gets deactivated?

Reply:
added